tests/test_cli.py

```diff
@@ -312,9 +312,10 @@ def test_folder(script_runner):
 
 def test_zipfile(script_runner):
     folder_name = "tests/testdata/folders/folder_one_file"
-    with tempfile.NamedTemporaryFile(suffix=".zip") as tmp:
-        create_zip(folder_name, tmp)
-        ret = script_runner.run('geoextent', '-b', '-t', tmp.name)
+    with tempfile.TemporaryDirectory() as tmp:
+        zip_path = tmp + "/zipfile.zip"
```

Comment:
This might break on some systems with different path formatting, please only use `os.path.join()`.